src/main/java/eu/europa/ted/efx/xpath/XPathScriptGenerator.java

```diff
@@ -471,6 +471,32 @@ public DurationExpression composeSubtraction(DurationExpression left, DurationEx
     return new DurationExpression("(" + left.script + " - " + right.script + ")");
   }
 
+
+  @Override
+  public <T extends Expression, L extends ListExpression<T>> L composeDistinctValuesFunction(
+      L listOne, L listTwo, Class<L> listType) {
+        return Expression.instantiate("distinct-values(" + listOne.script + ", " + listTwo.script + ")", listType);
```

Comment:
There's a set of parentheses missing here.
Currently this gives: `distinct-values(listOne, listTwo)`
This is not valid XPath, as distinct-values accepts only a single sequence.
I guess it shoud be: `distinct-values((listOne, listTwo))`